src/data_write.py

```diff
@@ -29,12 +29,9 @@
 from scipy.constants import speed_of_light
 import copy
 import pickle
-
-import utils.shared_macros as sm
 import utils.options.data_write_options as dwo
 from utils import socket_operations as so
 
-dw_print = sm.MODULE_PRINT("Data Write", "cyan")
 
 DATA_TEMPLATE = {
```

Comment:
If there was ever the perfect use case for a dataclass it was this. Making note of it for a future PR.

Reply:
Agreed. There are many places we can make that improvement.